<dec f='llvm/build/include/c++/v1/future' l='575' type='void std::__1::__assoc_sub_state::set_exception(std::exception_ptr __p)'/>
<def f='llvm/libcxx/src/future.cpp' l='110' ll='119' type='void std::__1::__assoc_sub_state::set_exception(std::exception_ptr __p)'/>
<use f='llvm/libcxx/src/future.cpp' l='210' u='c' c='_ZNSt3__17promiseIvED1Ev'/>
<use f='llvm/libcxx/src/future.cpp' l='239' u='c' c='_ZNSt3__17promiseIvE13set_exceptionESt13exception_ptr'/>
